src/Tree/Entity/Repository/NestedTreeRepository.php

```diff
@@ -825,6 +894,10 @@ public function verify()
         if (isset($config['root'])) {
             $trees = $this->getRootNodes();
             foreach ($trees as $tree) {
+                // if a root node is specified, verify only it
+                if (is_object($options['treeRootNode']) && $options['treeRootNode'] !== $tree) {
```

Comment:
If `$options['treeRootNode']` can only hold an object or `null`, I think `null !== $options['treeRootNode']` is cheaper.